api/python/t4/api.py

```diff
@@ -14,13 +14,39 @@
 
 
 def copy(src, dest, meta=None):
+    """
+    Retrieves ``src`` object from T4, and writes that to to the ``dest``
+    path.
+
+    Either of ``src`` and ``dest`` may be S3 paths (starting with ``s3://``)
+    or local file paths (starting with ``file:///``).
+
+    Parameters:
+        src (str): a path to retrieve
+        dest (str): a path to write to
+        snapshot (str): (optional) download from a specific snapshot
+        version (str): (optional) download a specific version
+    """
     all_meta = dict(
         user_meta=meta
     )
     copy_file(fix_url(src), fix_url(dest), all_meta)
 
 
 def put(obj, dest, meta=None):
+    """Write an in-memory object to the specified T4 ``dest``
+    Note:
```

Comment:
line break here. Usual style is one line description. Line break, paragraph description